samples/cellular/nrf_cloud_multi_service/boards/nrf9161dk_nrf9161_ns.conf

```diff
@@ -0,0 +1,18 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+# This file is merged with prj.conf in the application folder, and options
+# set here will take precedence if they are present in both files.
+
+# CAF - Common Application Framework
```

Comment:
I don't think MSS actually uses CAF.

I do see this was likely copied from the existing conf files, so the comment should probably be removed from those too

```
CONFIG_GPIO=y
CONFIG_LED_GPIO=y
````

are valid, just not related to CAF in this case.


I think I missed it back when this wording was first introduced, sorry about that